kili/cli.py

```diff
@@ -83,8 +89,12 @@ def list_project(api_key: str,
 @project.command(name='create')
 @api_key_option
 @endpoint_option
-@click.option('--interface', type=click.Path(exists=True), required=True,
-              help="Path pointing to your json interface file.")
+@click.option('--interface', type=str, required=True,
+              help=(
+                  "Path pointing to your json interface file."
+                  "of project_id of another Kili project"
```

Comment:
Path pointing to your json interface file or the project_id of another Kili project